readme.md

```diff
@@ -29,11 +29,19 @@ The plugin streamlines the entire configuration process and removes the need for
 - Jenkins 2.190.1+
 - Bitbucket Server 7.4+
 
-Note: Bitbucket Server 5.6 to 7.3 are also supported, but they're not recommended. This is because some plugin features are not available when using these versions. Instead, we recommend using Bitbucket Server 7.4+ as when you also set up an Application Link you will have access to all plugin features. 
+Note: Bitbucket Server 5.6 to 7.3 are also supported, but they're not recommended. This is because some plugin features are not available when using these versions. Instead, we recommend using Bitbucket Server 7.4+ so that you have access to all plugin features and you can set up an Application Link.
 
 ---
+## 1. Install the plugin
 
-## Configure the plugin
+To install:
+1. In Jenkins, go to **Jenkins** > **Manage Jenkins** > **Manage Plugins**.
+2. Under the **Available** tab, search for BitBucket Server Integration. 
+3. Tick the checkbox and select **Install without Restart**. 
```

Comment:
@ali3-atlassian do we need to give any more context around the checkbox? E.g.: Select the checkbox to ...

```suggestion
4. Tick the checkbox.
5. Select **Install without Restart**. 
```